.vscode/settings.json

```diff
@@ -1,6 +1,5 @@
 {
   "java.configuration.updateBuildConfiguration": "automatic",
-  "editor.formatOnSave": true,
```

Comment:
I removed this line while I was trying to get your stuff to format properly, but now please re-add it